core/common/src/main/java/alluxio/grpc/ServiceVersionClientServiceHandler.java

```diff
@@ -31,11 +32,16 @@
   /** Set of services that are going to be recognized by this versioning service. */
   private Set<ServiceType> mServices;
 
+  /**
+   * Creates service version handler that allows given services.
+   * @param services services to allow
```

Comment:
should there be a blank line before this?

Reply:
You mean a space between general doc and the @param doc? I can but it's pretty common through out the code.